src/assets.js

```diff
@@ -369,7 +395,7 @@ class Assets extends Events {
             data: options.data || api.schema.assets.getDefaultData('animstategraph'),
             folder: options.folder,
             preload: options.preload
-        });
+        }, null, options.onProgress);
```

Comment:
The default value for settings in `upload` is an empty object. Is null safe to pass here? If it is, the default value for `upload` should be null too.

Reply:
Good point - fixed.